frame/nomination-pools/src/lib.rs

```diff
@@ -1183,9 +1197,8 @@ impl<T: Config> BondedPool<T> {
 
 	/// Bond exactly `amount` from `who`'s funds into this pool.
 	///
-	/// If the bond type is `Create`, `Staking::bond` is called, and `who`
-	/// is allowed to be killed. Otherwise, `Staking::bond_extra` is called and `who`
-	/// cannot be killed.
+	/// If the bond type is `Create`, `Staking::bond` is called, and `who` is allowed to be killed.
+	/// Otherwise, `Staking::bond_extra` is called and `who` cannot be killed.
```

Comment:
```suggestion
	/// If the bond is [`BondType::Create`], [`Staking::bond`] is called, and `who` is allowed to be killed.
	/// Otherwise, [`Staking::bond_extra`] is called and `who` cannot be killed.
```